appveyor.yml

```diff
@@ -37,11 +40,9 @@ build_script:
   - bash build.sh
   - cd docs
   - bash builddocs.sh
-
-# TODO: Reinstate these...
-#  - bash runcoverage.sh
-#  - bash coveralls.sh
-
+  - cd ..
+  - bash runcoverage.sh
```

Comment:
We have "test: off" as tests run as part of build.sh already. We might want to change build.sh to accept a --no-test flag or similar so that we don't end up running all the tests twice... (I'm not sure whether we used to run all the tests twice or not. Back then there was less code to build...)